### PR TITLE
JDBC insert/update/delete

### PR DESCRIPTION
**Pull request by un-install**

### CODE REVIEW

File: src/main/java/lesson17/OfficeDAOImpl.java [hunk 1]
```diff
@@ -38,4 +39,52 @@
...
+    @Override
+    public boolean insertOffice(Offices office) throws SQLException {
+        Connection conn = OJDBCUtils.getConnection();
+        PreparedStatement stmt = conn.prepareStatement("insert into offices(OFFICE, CITY, REGION, MGR, TARGET, SALES)" +
+                " values (?, ?, ?, ?, ?, ?)");
+
+        setOfficeValuesToStatement(office, stmt);
+
+        boolean isOk = stmt.executeUpdate() > 0;
+        OJDBCUtils.closeAllCloseble(stmt, conn);
```
Comment (saadow) [inline src/main/java/lesson17/OfficeDAOImpl.java]: Cool...

File: src/test/java/lesson17/OfficeDAOTest.java [hunk 2]
```diff
@@ -22,4 +23,21 @@ public void testGetOfficesByRegion() throws SQLException {
...
+    @Test
+    public void testUpdateOffice() throws SQLException {
+        assertTrue(odao.updateOffice(new Offices(new BigDecimal(1233), "Miami", "Восток",
+                new BigDecimal(103), null, new BigDecimal(1111))));
+    }
+
+    @Test
+    public void testDeleteOffice() throws SQLException {
+        assertTrue(odao.deleteOffice(new BigDecimal(1233)));
+    }
```
Comment (saadow) [inline src/test/java/lesson17/OfficeDAOTest.java]: how about FixMethodOrder annotation?
https://github.com/junit-team/junit4/wiki/test-execution-order

File: src/test/java/lesson17/OfficeDAOTest.java [hunk 3]
```diff
@@ -22,4 +26,21 @@ public void testGetOfficesByRegion() throws SQLException {
         Set<Offices> actual = odao.getOfficesByRegion("Запад");
         assertTrue(actual.size() > 0);
     }
+
+    @Test
```
Comment (SergeyPasko) [inline src/test/java/lesson17/OfficeDAOTest.java]: Start full test class. Some of your methods will be red (depends to order)

File: src/main/java/lesson17/OfficeDAOImpl.java (hunk 1 above)
Comment (SergeyPasko) [reply src/main/java/lesson17/OfficeDAOImpl.java]: It isn't help you if we have exception. try{}finally{}

File: src/main/java/lesson17/OfficeDAOImpl.java [hunk 4]
```diff
@@ -41,16 +41,22 @@
...
-        return isOk;
+        PreparedStatement stmt = null;
+        boolean isOk = false;
+        try {
+            stmt =  conn.prepareStatement("insert into offices(OFFICE, CITY, REGION, MGR, TARGET, SALES)" +
+                    " values (?, ?, ?, ?, ?, ?)");
+            setOfficeValuesToStatement(office, stmt);
+
+            isOk = stmt.executeUpdate() > 0;
+        }catch (SQLException e){
```
Comment (SergeyPasko) [inline src/main/java/lesson17/OfficeDAOImpl.java]: As variant =)

**Pull request merged**